### PR TITLE
add website packages

### PR DESCRIPTION
**Pull request by thuntran**

### CODE REVIEW

**Comment by thuntran** [comment]
@Chenhui-Jia Can you help me double check if the packages (and their versions) are correct/whether I'm missing anything? I just `pip show sphinx` in the terminal but not sure if I'm missing anything.

**Review by mariumtapal — APPROVED** [review]

**Comment by mariumtapal** [comment]
@thuntran solved the merge conflict! to avoid more merge conflicts, pull before committing any new changes!😅

**Comment by thuntran** [comment]
> @thuntran solved the merge conflict! to avoid more merge conflicts, pull before committing any new changes!😅

Yeah I think I forgot to pull before committing, sorry about that! And thanks for helping with resolving the merge conflict :)

**Comment by mariumtapal** [comment]
> > @thuntran solved the merge conflict! to avoid more merge conflicts, pull before committing any new changes!😅
> 
> Yeah I think I forgot to pull before committing, sorry about that! And thanks for helping with resolving the merge conflict :) @mariumtapal

@thuntran Oh no you're doing fine - I made a commit to your branch that's why I just mentioned it :)

File: setup.py [hunk 1]
```diff
@@ -34,12 +34,12 @@
...
             'sphinx != 1.3.1',
-            'nbsphinx',
-            'sphinx_rtd_theme==0.5.*',
-            'sphinx-multiversion >= 0.2.3',
-            'sphinx-gallery >= 0.7',
-            'spinxcontrib-svg2pdfconverter',
-            'presets'
+            'sphinx_rtd_theme == 0.5.*',
+            'nbsphinx == 0.8.*',
+            'spyder == 4.2.*',
```
Comment (Chenhui-Jia) [inline setup.py:39]: my spyder is 4.0.1, my numpydoc is 0.9.2,  so we can probably lower the version?

**Review by Chenhui-Jia — APPROVED** [review]

**Pull request merged**